holospy/reconstruct.py

```diff
@@ -25,18 +25,18 @@
 
 
 def estimate_sideband_position(
-    holo_data, holo_sampling, central_band_mask_radius=None, sb="lower", high_cf=True
+    data, sampling, central_band_mask_radius=None, sb="lower", high_cf=True
```

Comment:
The proposed renaming of parameters does make sense after the split, but shouldn't it be mentioned explicitly in the changelog?

Reply:
Done!